generator/model.go

```diff
@@ -543,8 +543,23 @@ func hasValidations(model *spec.Schema, isRequired bool) (needsValidation bool,
 	return
 }
 
+// handleFormatConflicts handles all conflicting model properties when a format is set
+func handleFormatConflicts(model *spec.Schema) {
+	switch model.Format {
+	case "date":
+		model.MinLength = nil
+		model.MaxLength = nil
+	case "datetime":
```

Comment:
Since "date" and "datetime" have the same conflicts, you can combine the cases:

    case "date", "datetime":

Reply:
Probably "uuid" as well.